packages/ui-components/src/index.ts

```diff
@@ -10,3 +10,4 @@ export * from "./ExpandableText/ExpandableText";
 export * from "./Spinner/Spinner";
 export * from "./Typography/";
 export * from "./Logo/";
+export * from "./Charts/barChart"; 
```

Comment:
You are missing a new line here, which is causing linting to fail.

Reply:
ty!